### DISCUSSION TITLE
[BUG]A SCAM is a SCAM is a SCAM

### DISCUSSION THREAD

**Discussion by raz0rsedge1975** [Chia-Network/chia-blockchain#3594] [General]
Don`t people ever get rid of the next crypto coin scam? 
I wonder what comes next...prooff of greediness and blunder?
What`s THIS green about it? The burnin of megawatts of power for the next greed?

LMAO

**Comment by Leozolt** [comment]
lol

**Comment by PCDSP** [comment]
Chia Network has moved this to a discussion! 

The one green thing about it is the leaf logo!

At 1PiB of data (1000TB) you stand to earn 1-2 XCH/ day... Everyone can do this with some 50000$ worth of gear and 1000s/mo maintenance and expenses! 

This pothead Bram is amazing at marketing! Soon Elon will show as a video of how they came up with Chia!

**Comment by pluto2021** [comment]
According to some earlier comments, the thought-to-be probability of winning was not bad based on some 200-400PiB spaces when the number of rewards per day is nailed.  I am not sure how the original network growth model compares with the real growth rate so far.  The network is growing exponentially at a formidable rate, while anyone's plotting capability is pretty much fixed. Within a month or so, the net space will out pace any small farms and it would no longer make any sense to invest in any new plotting rigs, as the time it takes to recover the investment can be indefinite.  This might be the time when the actual net space growth would slow down. So the Chia concept largely depends on the pools.  Without the ability to pool, you are literally buying very expensive lottery tickets that also carries a maintenance fee. 

The original concept of using existing HDD space is largely flawed, as the greed would come in trying to get on the bus before too late. People end up buying HDD spaces they might never need for their entire life + new plotting rigs. It is this in rush due to human greed fueled the explosive growth of net space.

**Reply by PCDSP** [reply]
Bram bought shares in all HDD producing manufacturers.

He is also a true champion of ecology with PiB of 10% failure rate hdds and 24/7 running of said PiB of storage! 

Truly green as the logo shows!

**Reply by pluto2021** [reply]
Besides, when the 10% failure rate would hit any individual space farmer, it can wipe out significant portion of the HDD space he would have owned. One HDD fails, the entire collection of plots may render worthless, as the cost to recover the data may far exceed the profit they may generate in any meaningful time.

**Reply by Leozolt** [reply]
With the actual price of chia it would make sense to invest in tape storage to back up the plots. So if a HDD fails it is easily recovered. lol

**Comment by PCDSP** [comment]
lol lol that will be eco-friendly! you would not have to write 15x the amount of a plot again, at high wattage CPU and NVME burnout to rebuild them! 

Should be mentioned in the 101 manual of best practice, to keep the coin green! 

Bunch of muppets!

**Comment by fullet-tortuga** [comment]
I can think of proof of fan speed, proof of monitor brightness, proof of speakers volume... the possibilities are endless X-D Sad thing is that people will pile up again on whatever you make up.

**Reply by Leozolt** [reply]
If you look back at the market from last year.. Rumor tells that there is something in preparation called proof of toilet paper and early birdiness?

**Reply by PCDSP** [reply]
Let's buy shares first then make a crypto for proof of AiO Pump SPEED! :)
